assets/acquia_lift.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file
+ * Contains the definition of the behaviour openAcquiaLiftExperienceBuilder.
+ */
+
+(function ($, Drupal) {
+
+    'use strict';
+
+    /**
+     * Attaches the JS test behavior to to weight div.
+     */
+    Drupal.behaviors.acquiaLiftOpenExperienceBuilder = {
+        attach: function (context, settings) {
+            $('.openLiftLink').click(function() {
```

Comment:
`.click()` is deprecated in jQuery. Use `.on('click')`, just like Drupal core does. Also, why jQuery? You can do this without.